src/jobs/PkgEvalJob.jl

```diff
@@ -871,7 +878,7 @@ function get_history(cfg, days = 30)
     end
 
     # Convert the dict into a string representations
-    heading = "History ($start_date to $end_date)"
+    heading = "History (last 30 days)"
```

Comment:
I think having the exact dates is useful because it allows precise cross-referencing between reports as well as making it easier to identify the exact date a package or group of packages started to fail.

If length is a significant problem, we could cut out the year field
```
History (2023-07-14 to 2023-08-12)
History (07-14 to 08-12)
History (last 30 days)
```

On chrome, with a non-monospace font, the header is substantially narrower than the history field.
<img width="870" alt="Screenshot 2023-08-15 at 6 44 27 AM" src="https://github.com/JuliaCI/Nanosoldier.jl/assets/60898866/2bffd82f-43ca-4015-b445-81422b86efc5">

Reply:
Yeah, the monospace fonts narrows it significantly (but removes the spacing between characters I was seeing with Firefox' default font).



> I think having the exact dates is useful because it allows precise cross-referencing between reports

The dates should be mentioned at the top already, right?

I also thought we could make the tooltip show the date, but the regex approach you took doesn't make that trivial to add, so I went with this approach first.